tests/constants/session.constants.ts

```diff
@@ -21,10 +21,5 @@ export const NO_OPEN_PHASES_SESSION_DATA: SessionData = {
 
 export const MULTIPLE_OPEN_PHASES_SESSION_DATA: SessionData = {
   ...BUG_REPORTING_PHASE_SESSION_DATA,
-  openPhases: [
-    Phase.phaseBugReporting,
-    Phase.phaseTeamResponse,
-    Phase.phaseTesterResponse,
-    Phase.phaseModeration
-  ]
+  openPhases: [Phase.phaseBugReporting, Phase.phaseTeamResponse, Phase.phaseTesterResponse, Phase.phaseModeration]
```

Comment:
Is this from a prettier fix? Maybe we should leave it out since its not related to the feature in this PR.

Reply:
Yes, we accidentally missed some test code when adopting prettier (due to the ongoing PRs at that time).
We can open a separate PR for this